Lorenzo0111/ElectionsPlus
# Dependency updates, message cleanup, bugfixes, and new features

* Upgrade Gradle
* Update dependencies to latest versions so it works on Spigot & Paper 1.19.4
* Cleanup and normalize all messages to use MiniMessage formatting
  * Get rid of translateAlternateColorCodes
  * Add more (most?) messages to messages.yml
* Remove sponge support (testing is out of scope for me)
* Add new commands & subcommands so parties & elections can be managed via "/" commands
* Add new feature to open the elections menu when a player hits a block
* bugfix: Table creation happens asynchronously, so SQL errors ('table not found') errors are raised if code tries to query the table and it hasn't yet been created (see issue #125)
